src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final String lessThan6Chars = "oops";
+    private final String over6Chars = "misterProper";
+    private final int minAge = 18;
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+    private User testNullUser = new User();
+    private User testUser2 = new User();
+    private StorageDao storageDao = new StorageDaoImpl();
+

```

Comment:
consider using @BeforeAll or @BeforeEach for 
1 Class instantiation
2 Creating the valid test data (and change it through setters in case of necessarily)

fields e.g. minAge should be or constant, or local vars.